apps/site/assets/ts/stop/__tests__/DepartureListTest.tsx

```diff
@@ -181,4 +188,35 @@ describe("DepartureList", () => {
     );
     expect(screen.getByText("No upcoming trips today")).toBeDefined();
   });
+
+  it("should display cancelled if the trip has been cancelled", () => {
+    jest.spyOn(predictionsChannel, "default").mockImplementation(() => {
+      return {
+        "TestRoute Route": [
+          {
+            time: new Date("2022-04-27T11:15:00-04:00"),
+            trip: { id: "1" },
+            schedule_relationship: "cancelled"
+          }
+        ] as PredictionWithTimestamp[]
+      };
+    });
+
+    const schedules = [
+      {
+        trip: { id: "1", headsign: "TestRoute Route" }
+      }
+    ] as ScheduleWithTimestamp[];
+
+    render(
+      <DepartureList
+        alerts={[]}
+        route={route}
+        stop={stop}
+        schedules={schedules}
+        directionId={0}
+      />
+    );
+    expect(screen.getByText("Cancelled")).toBeInTheDocument();
```

Comment:
can a line be added to test that the time text is struck thru as well?

Reply:
I struggle with that.  Testing library doesn't support testing for styling. I feel like this is intentional as they view that level of testing as too tedious, or getting into the implementation details instead of just the component logic.  There are sure exceptions where we set only styling based on conditional logic (which is why we can breakout of testing library to test for it).  I don't think this is one of the exceptions.  All the styling and wording come bundled together in out implementation so testing for just the word should be sufficient to test that the desired state is reached.